9.tcga-classify/scripts/nbconverted/visualize-results.r

```diff
@@ -15,53 +15,54 @@ full_mutation_df <- load_results(results_path = mut_path, process_output = TRUE)
 cancertype_path <- file.path("results", "cancer-type")
 full_cancertype_df <- load_results(results_path = cancertype_path, process_output = TRUE)
 
-# Setup plotting logic for supplmental plot
-genes <- unique(full_mutation_df$gene_or_cancertype)
-
-top_25 <- full_mutation_df %>%
-  dplyr::filter(gene_or_cancertype %in% genes[1:25])
-bottom_25 <- full_mutation_df %>%
-  dplyr::filter(gene_or_cancertype %in% genes[26:length(genes)])
+# Setup plotting logic for supplmental plots
+genes <- unique(levels(full_mutation_df$gene_or_cancertype))
+cancertypes <- as.character(unique(full_cancertype_df$gene_or_cancertype))
 
 # Create mutation plots
-top25_gg <- plot_mutation_figure(df = top_25)
-bottom25_gg <- plot_mutation_figure(df = bottom_25)
+gg_list <- list()
+for (plot_idx in seq(1, 50, 10)) {
```

Comment:
Nit picky -- might want to assign variable names instead of hard coding a constant so you know what those numbers mean

Reply:
good point. Will update! Thanks!